### PR TITLE
Adds scss support to website 

### PR DESCRIPTION
**Pull request by ry**
(reduces number of requests to load style sheets from 5 to 1)

### CODE REVIEW

File: website/main.scss [hunk 1]
```diff
@@ -1,5 +1,10 @@
+@import './normalize.scss';
+@import './skeleton.scss';
+@import './codemirror.scss';
+@import './syntax.scss';
 @import url('https://fonts.googleapis.com/css?family=Roboto|Roboto+Mono');
 
+
```
Comment (piscisaureus) [inline website/main.scss]: ...?

**Comment by piscisaureus** [comment]
why is normalize.css duplicated, while the others are renamed?

File: tools/build_website.ts [hunk 2]
```diff
@@ -62,13 +63,26 @@ async function writePages() {
   }
 }
 
+function scss(inFile, outFile) {
+  const options = {
+    file: inFile,
+    includePaths: ["./website"],
+  };
+
+  const result = renderSync(options).css.toString("ascii");
```
Comment (piscisaureus) [inline tools/build_website.ts]: why ascii and not utf8?

File: website/main.scss (hunk 1 above)
Comment (ry) [reply website/main.scss]: Fixed.

File: tools/build_website.ts (hunk 2 above)
Comment (ry) [reply tools/build_website.ts]: Fixed.

**Comment by ry** [comment]
normalized fixed too.

**Comment by piscisaureus** [comment]
lgtm

**Pull request merged**